core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -44,8 +44,10 @@
   public static final String ICEBERG_CATALOG_TYPE = "type";
   public static final String ICEBERG_CATALOG_TYPE_HADOOP = "hadoop";
   public static final String ICEBERG_CATALOG_TYPE_HIVE = "hive";
+  public static final String ICEBERG_CATALOG_TYPE_JDBC = "jdbc";
   public static final String ICEBERG_CATALOG_HIVE = "org.apache.iceberg.hive.HiveCatalog";
   public static final String ICEBERG_CATALOG_HADOOP = "org.apache.iceberg.hadoop.HadoopCatalog";
+  public static final String ICEBERG_CATALOG_JDBC = "org.apache.iceberg.hadoop.HadoopCatalog";
```

Comment:
Shouldn't this refer to the `JdbcCatalog` class not `HadoopCatalog`?

Reply:
corrected it